### PR TITLE
module/*/ignition.tf: fix regression in ignition files

### PR DESCRIPTION
**Pull request by kalmog**

### CODE REVIEW

File: modules/openstack/etcd/ignition.tf [hunk 1]
```diff
@@ -5,12 +5,11 @@ data "ignition_config" "etcd" {
     "${data.ignition_user.core.id}",
   ]
 
-  files = ["${compact(list(
```
Comment (enxebre) [inline modules/openstack/etcd/ignition.tf]: what's the motivation behind removing `compact(list(` ?

**Comment by kalmog** [comment]
ok to test

File: modules/openstack/etcd/ignition.tf (hunk 1 above)
Comment (kalmog) [reply modules/openstack/etcd/ignition.tf]: the `compact(list)` wasn't there to begin with and was accidentally introduced in #2714. This PR rectifies that.

Comment (lander2k2) [reply modules/openstack/etcd/ignition.tf]: The original PR [1] actually introduced that `compact(list(` change.

[1] https://github.com/coreos/tectonic-installer/pull/2609/files#diff-92eb4aaccf04300684240249801f91f2

**Comment by lander2k2** [comment]
@kalmog I was just comparing the commit history on your branch [1] vs mine[2].  The content of the commits are different.  It looks like changes were omitted somewhere.  I lost count of the rebases I did on that original PR - perhaps you cherry-picked old or mismatched commits?

[1] https://github.com/kalmog/tectonic-installer/commits/vmw_track-1_http_proxy
[2] https://github.com/lander2k2/tectonic-installer/commits/vmw_track-1_http_proxy

**Pull request closed without merging**